uibackend/server/go.mod

```diff
@@ -0,0 +1,75 @@
+module github.com/openclarity/vmclarity/uibackend/server
+
+go 1.21.4
+
+replace (
```

Comment:
`replace` directives should be added after the `require` directives, at the bottom of the file in this case.